recipes/blackboxauditing/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "BlackBoxAuditing" %}
+{% set version = "0.1.54" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 02824682f4ac3a89e606d3dc7e99481d09e82655513511e9406268f8b17be3d6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

I think this recipe could be ``noarch``
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

---

adding `noarch: python` under the build section is the way to do this right?

---

Yep!